index.html

```diff
@@ -4,7 +4,7 @@
         <body>
             <head>
```

Comment:
Please always include the HTML boilerplate. This is an essential part of ALL HTML documents.
You can read more about it here.
(https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/#:~:text=According%20to%20Wikipedia%2C,all%20of%20your%20HTML%20pages.)